components/Sidebar.tsx

```diff
@@ -1,245 +1,93 @@
-import { useCallback, useEffect, useState } from 'react'
+import Brand from 'assets/brand.svg'
+import clsx from 'clsx'
 import { useWallet } from 'contexts/wallet'
-import type { NextPage } from 'next'
-import Link from 'next/link'
-import Image from 'next/image'
-import { ImArrowUpRight2, ImGithub, ImTwitter } from 'react-icons/im'
-import { FiMoon, FiSun, FiBox } from 'react-icons/fi'
-import { SiDiscord, SiTelegram } from 'react-icons/si'
-import { BiWallet } from 'react-icons/bi'
-import { useTheme } from 'contexts/theme'
-import getShortAddress from 'utils/getShortAddress'
-import { loadKeplrWallet, useKeplr } from 'services/keplr'
 import { useRouter } from 'next/router'
-import { getConfig } from 'config'
+import { footerLinks, links, socialsLinks } from 'utils/links'
 
-const Sidebar: NextPage = () => {
-  const router = useRouter()
-  const theme = useTheme()
-  const wallet = useWallet()
-  const keplr = useKeplr()
-
-  const activeColor = theme.isDarkTheme ? 'bg-purple/25' : 'bg-purple/10'
-  const walletText = wallet.initialized
-    ? wallet.name || getShortAddress(wallet.address)
-    : 'Connect Wallet'
-
-  const changeThemeOnClick = () => {
-    theme.setIsDarkTheme(!theme.isDarkTheme)
-  }
+import Anchor from './Anchor'
+import SidebarContainer from './SidebarContainer'
+import SidebarLayout from './SidebarLayout'
+import SidebarToggle from './SidebarToggle'
```

Comment:
@XanderJL, here's my attempt splitting to separate components as you recommended.

Reply:
Awesome! Zip it and ship it 😎 🚀